Library/Homebrew/download_strategy.rb

```diff
@@ -180,6 +180,16 @@ def extract_ref(specs)
 
 class AbstractFileDownloadStrategy < AbstractDownloadStrategy
   def stage
+    wd = Dir.pwd
+    unless (wd == HOMEBREW_CELLAR.to_s)
+      cellar_gid = File.stat(HOMEBREW_CELLAR).gid
+      wd_gid = File.stat(wd).gid
+      if (cellar_gid != wd_gid)
+        puts("#{wd} gid was #{wd_gid}")
+        FileUtils.chown(nil, cellar_gid, wd)
+        puts("#{wd} gid is now #{File.stat(wd).gid}")
```

Comment:
I think you can skip the `puts` here (or at least when `ARGV.verbose?` is unset)